Src/Support/Google.Apis.Auth/OAuth2/LocalServerCodeReceiver.cs

```diff
@@ -154,27 +165,31 @@ public class ServerException : Exception
                 public ServerException(string msg) : base(msg) { }
             }
 
-            public static LimitedLocalhostHttpServer Start(string url)
+            public static LimitedLocalhostHttpServer Start(string url, string _closePageResponse)
```

Comment:
Please remove the `_` prefix on the parameter name. A `_` prefix is for fields only.